lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3597,6 +3604,14 @@ protected function get_feature_declarations_for_node( $metadata, &$node ) {
 				// to leverage existing `compute_style_properties` function.
 				$feature_node = array( $feature => $node[ $feature ] );
 
+				// If the feature is a state selector (e.g. `@currentItem`)
```

Comment:
I wonder if we should consider current item a state? It's not a state really (like say hover is), it's just a special selector.

Reply:
Yeh we need to get the nomenclature for this right now. If we put "state" in the code the feature will be known as "state".

Still...I can't think of a better term.

I asked an AI assistance and it came up with

````
"It's currently in use."
"It's presently active."
"It's in operation."
"It's functioning now."
"It's currently live."
"It's currently enabled."
"It's in effect."
"It's the current choice."
"It's presently selected."
"It's the current option."
```

Reply:
> what is the name of core/navigation node in theme json? that is the same as core/navigation > currentItem

Sorry I don't understand what you are asking 🙏

Reply:
How do we refer to nodes like `core/navigation` (a block name)  in theme json, what kind of node is it? (style, properties?). Becasue the `@current` node is an override of the parent - not a state.